kedro/framework/project/__init__.py

```diff
@@ -272,8 +272,12 @@ def validate_settings():
             "'bootstrap_project'. This should happen automatically if you are using "
             "Kedro command line interface."
         )
-
-    importlib.import_module(f"{PACKAGE_NAME}.settings")
+    # Check if file exists, if it does, validate it.
+    if importlib.util.find_spec(f"{PACKAGE_NAME}.settings") is not None:  # type: ignore
+        importlib.import_module(f"{PACKAGE_NAME}.settings")
+    else:
+        logger = logging.getLogger(__name__)
+        logger.warning("No settings.py found, defaults will be used.")
```

Comment:
Minor suggestion to get nicer logs highlighting: 
```suggestion
        logger.warning("No 'settings.py' found, defaults will be used.")
```